src/modules/universal-discovery/components/finder/finder.js

```diff
@@ -1,14 +1,25 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useRef } from 'react';
 
 import FinderBranch from './finder.branch';
 
 import { LoadedLocationsMapContext } from '../../universal.discovery.module';
 
 const Finder = () => {
     const [loadedLocationsMap, dispatchLoadedLocationsAction] = useContext(LoadedLocationsMapContext);
+    const finderRef = useRef(null);
+
+    useEffect(() => {
+        const container = finderRef.current;
+
+        if (!container) {
```

Comment:
Is this if needed? useEffect is called each time after page is rendered, so this `<div>` will be always created.